Dockerfile

```diff
@@ -5,19 +5,20 @@ RUN pip install --upgrade pip
 COPY requirements.txt /requirements.txt
 RUN pip install -r /requirements.txt
 #FROM python:2-onbuild
-RUN apt-get install curl
+RUN apt-get install curl=7.52.1-5+deb9u6 --no-install-recommends
```

Comment:
isn't this needs to be like `apt-get update && apt-get install curl `

Reply:
Usually just need to do an update if the repositories change and to look for newer versions. 

By not running an update here the version really isn't needed at all. 

But later in the script I do update below to pickup the new yarn sources which will update the other packages there. 

This should probably be all re-organized, I'll update that all in the other PR when we switch from apt to apk.